### PR TITLE
Proxy set up v1

### PR DESCRIPTION
**Pull request by masoncpott**
@sam-masno @sjamcox @tyler-thee-creator 

I set up the most basic version of a reverse proxy just to get everything connected and working.
Next, I need to add more routes to the API and then hook it up to new relic, add Artillery configuration, and then conduct stress tests. I plan on doing all of that tonight in class.

### CODE REVIEW

File: server/host.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+module.exports = {
```
Comment (tyler-thee-creator) [inline server/host.js:1]: Really like how you have host as a separate variable for where you're reaching out to the service, clean and allows you to easily change all in one place, I'll have to try and implement this on my proxy

File: server/index.js [hunk 2]
```diff
@@ -12,4 +14,14 @@ app.use(cors());
 app.use(express.static(`${__dirname}/../public`));
 app.use('/:id', express.static(`${__dirname}/../public`));
 
+app.get('/priceandinventory/id/:productId', async (req, res) => {
```
Comment (tyler-thee-creator) [inline server/index.js:17]: Nice and clean route, good job

**Review by tyler-thee-creator — APPROVED** [review]
Looks great, nicely done

File: server/host.js (hunk 1 above)
Comment (masoncpott) [reply server/host.js:1]: Yeah the idea is that it will make updating the URL's easier when deployed. I got this idea from Sam.

**Pull request merged**